app/controllers/concerns/foreman_webhooks/controller/parameters/webhook.rb

```diff
@@ -26,6 +26,7 @@ def webhook_params_filter
         end
 
         def webhook_params
+          params['webhook']['event'] = params['event'] unless params['event'].blank?
```

Comment:
This should fix the error `events can't be blank` even if it was selected in the form. Not sure why it doesn't get into permitted params.